commands/doctor.cmd

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+[[ ! ${WARDEN_DIR} ]] && >&2 echo -e "\033[31mThis script is not intended to be run directly!\033[0m" && exit 1
+
+# The warden doctor command is designed to collect information useful in reasoning about the
+# state of a system and configuration warden is running with.
+
+# Host information
+## darwin
+# > sw_vers
+
+if [[ $OSTYPE =~ ^darwin ]]; then
+    echo -e "\033[032mHost information:\033[0m"
+    sw_vers --productName
+    sw_vers --productVersion
+fi
+echo
+
+echo -e "\033[32mOS, version, architecture:\033[0m"
+uname -orm
+echo
+
+echo -e "\033[32mHomebrew information:\033[0m"
+brew config
```

Comment:
I have concerns with how this acts if brew doesn't exist

Reply:
Brew is a pre-requisite of Warden, no?

Reply:
It is not, no. It is recommended but not required.

Reply:
OK, I will add a check and guard around that block. 

Technically it is listed as a requirement on the warden.dev site. `brew install mutagen` is also called directly in the sync command.

<img width="1216" alt="image" src="https://github.com/user-attachments/assets/0b54527a-b831-4ca5-b241-32df84fc337e" />

If you wanted to say `jq` is likely to be unsafe that is possible. I will add a check for that as well though there are lots of assumptions of commands being available throughout Warden. Possibly we should add an issue to audit them and come up with a way to verify them as part of this command.

Reply:
Documentation says something different. I think the site just needs to be adjusted to remove the homebrew requirement, and make it the preferred installation method.
<img width="738" alt="image" src="https://github.com/user-attachments/assets/12915986-2662-42a7-8670-8caf36e08909" />